app/controllers/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -78,6 +78,15 @@ def reassign_hostgroup
         end
       end
 
+      api :PUT, "/hosts/bulk/change_owner", N_("Change owner")
+      param_group :bulk_host_ids
+      param :owner_id, :number, :required => true, :desc => N_("ID of the owner to reassign the hosts to")
+      def change_owner
+        BulkHostsManager.new(hosts: @hosts).change_owner(params[:owner_id])
+        process_response(true, { :message => n_("Updated %{count} host: changed owner",
+          "Updated %{count} hosts: changed owner", @hosts.count) % {count: @hosts.count} })
```

Comment:
There is a concern that `@host.count` may not be the actual number of hosts that got updated. 
Maybe:
````
process_response(true, { :message => n_("Updated host: changed owner", "Updated hosts: changed owner", @hosts.count)})